package.json

```diff
@@ -21,6 +21,10 @@
   "bugs": {
     "url": "https://github.com/hubotio/hubot-help/issues"
   },
+  "dependencies": {
+    "moment": "^2.22.2",
+    "hubot-routines": "https://github.com/tolstoyevsky/hubot-routines.git"
+  },
```

Comment:
There is the same question as upper.

Reply:
Yes, there was a need for a separate module to check the administrator